src/sentry/taskworker/client.py

```diff
@@ -10,19 +11,35 @@
 )
 from sentry_protos.taskbroker.v1.taskbroker_pb2_grpc import ConsumerServiceStub
 
+from sentry.utils import json
+
 logger = logging.getLogger("sentry.taskworker.client")
 
+SERVICE_CONFIG = json.dumps(
+    {
+        "loadBalancingConfig": [{"round_robin": {}}],
+    }
+)
+GRPC_OPTIONS = [
+    ("grpc.service_config", SERVICE_CONFIG),
+]
+
 
 class TaskworkerClient:
     """
     Taskworker RPC client wrapper
     """
 
     def __init__(self, host: str) -> None:
+        global GRPC_OPTIONS
         self._host = host
 
         # TODO(taskworker) Need to support xds bootstrap file
-        self._channel = grpc.insecure_channel(self._host)
+        grpc_override = os.getenv("SENTRY_TASKWORKER_GRPC_SERVICE_CONFIG")
+        if grpc_override:
+            GRPC_OPTIONS = [("grpc.service_config", grpc_override)]
```

Comment:
You'll mutate globals here. Perhaps we could have a local alias that is mutated?

```python
options = env("SENTRY_TASKWORKER_GRPC_SERVICE_CONFIG", SERVICE_CONFIG)
```